### PR TITLE
Markup test cases

### PR DESCRIPTION
**Pull request by jasollien**

### CODE REVIEW

File: Test Cases/v3.0/Markup/Document Reference External/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## Testing process
+
+1. Import _DocumentReferenceExternal.bcf_.
+2. Verify the bcf was imported correctly:
+> 1. Your application is aware of the absolute reference to the document outside of the BCFzip as given in the _DocumentReferences_ property in the _markup.bcf_ file.
+> 2. The .xsd file attachment can be retrieved (public download from buildingsmart-tech.org).
+
+3. Export the topic you imported to _exported.bcf_.
+4. Verify that no information was lost during the export
```
Comment (leosobral) [inline Test Cases/v3.0/Markup/Document Reference External/README.md:13]: missing dot at end of sentence

File: Test Cases/v3.0/Markup/Document Reference External/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+---
```
Comment (leosobral) [inline Test Cases/v3.0/Markup/Document Reference External/README.md:15]: Remove those lines below ..

File: Test Cases/v2.1/Markup/Document Reference External/unzipped/extensions.xsd [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        <TopicStatus>Open</TopicStatus>
+        <TopicStatus>Closed</TopicStatus>
+    </TopicStatuses>
+    <Priorities/>
+    <TopicLabels>
+        <TopicLabel>Architects</TopicLabel>
+        <TopicLabel>Engineers</TopicLabel>
+    </TopicLabels>
+    <Users>
+        <User>jon.anders.sollien@catenda.no</User>
```
Comment (leosobral) [inline Test Cases/v2.1/Markup/Document Reference External/unzipped/extensions.xsd:19]: remove these real test users references

File: Test Cases/v3.0/Markup/Due Date/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Due Date
+
+This test case verifies support of the `DueDate` property on a topic.
+
+## Testing process
+
+1. Import _DueDate.bcf_.
```
Comment (leosobral) [inline Test Cases/v3.0/Markup/Due Date/README.md:7]: Missing dots at end of all sentences for consistency

File: Test Cases/v3.0/Markup/related topics with both topics in the same file/README.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# \<Related topics with both topics in the same file\>
+
+This test case verifies support for related topics
+
+## Testing process
+
+1. Import \case.bcf
```
Comment (leosobral) [inline Test Cases/v3.0/Markup/related topics with both topics in the same file/README.md:7]: Missing dots at end of bullet items

File: Test Cases/v3.0/Markup/related topics with both topics in the same file/README.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+2. Verify the bcf was imported correctly:
+
+> 1. Check that the topic has a related topic, and that is available
+
+> 
+> Verification paragrpah 2
```
Comment (leosobral) [inline Test Cases/v3.0/Markup/related topics with both topics in the same file/README.md:13]: typo on paragraph

**Review by leosobral — CHANGES_REQUESTED** [review]

File: Test Cases/v3.0/Markup/Document reference internal/unzipped/8ac9822a-761a-4deb-9f39-f61286acbf6a/markup.bcf [hunk 7]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Markup xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="markup.xsd">
+    <Header>
+        <Files/>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Markup/Document reference internal/unzipped/8ac9822a-761a-4deb-9f39-f61286acbf6a/markup.bcf:4]: Files is empty.

File: Test Cases/v3.0/Markup/Due Date/unzipped/fffc1b9d-1f64-46ee-ad84-4fd4a0640e5f/markup.bcf [hunk 8]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Markup xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="markup.xsd">
+    <Header>
+        <Files/>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Markup/Due Date/unzipped/fffc1b9d-1f64-46ee-ad84-4fd4a0640e5f/markup.bcf:4]: Files is empty

File: Test Cases/v3.0/Markup/Labels/unzipped/bee19eb8-3ec0-4e0d-90df-52afc806beaf/markup.bcf [hunk 9]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Markup xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="markup.xsd">
+    <Header>
+        <Files/>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Markup/Labels/unzipped/bee19eb8-3ec0-4e0d-90df-52afc806beaf/markup.bcf:4]: Files is empty

File: Test Cases/v3.0/Markup/Milestone/unzipped/547bb53e-0c84-4a07-a75d-19a68576394c/markup.bcf [hunk 10]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Markup xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="markup.xsd">
+    <Header>
+        <Files/>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Markup/Milestone/unzipped/547bb53e-0c84-4a07-a75d-19a68576394c/markup.bcf:4]: Files is empty

File: Test Cases/v3.0/Markup/User assignment/unzipped/7ad1a717-bf20-4c12-b511-cbd90370ddba/markup.bcf [hunk 11]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Markup xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="markup.xsd">
+    <Header>
+        <Files/>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Markup/User assignment/unzipped/7ad1a717-bf20-4c12-b511-cbd90370ddba/markup.bcf:4]: Files is empty.

File: Test Cases/v3.0/Markup/related topics with both topics in the same file/unzipped/a6f801b9-6bf6-4cb9-8b89-1ae24b76074a/markup.bcf [hunk 12]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Markup xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="markup.xsd">
+    <Header>
+        <Files/>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Markup/related topics with both topics in the same file/unzipped/a6f801b9-6bf6-4cb9-8b89-1ae24b76074a/markup.bcf:4]: Files is empty

**Comment by pasi-paasiala** [comment]
extensions.xsd need to renamed to extensions.xml

File: Test Cases/v3.0/Markup/Due Date/unzipped/fffc1b9d-1f64-46ee-ad84-4fd4a0640e5f/markup.bcf (hunk 8 above)
Comment (jasollien) [reply Test Cases/v3.0/Markup/Due Date/unzipped/fffc1b9d-1f64-46ee-ad84-4fd4a0640e5f/markup.bcf:4]: Isn't that allowed?

**Pull request merged**